backend/tests/test_crud_schema.py

```diff
@@ -691,14 +629,17 @@ def test_raise_on_multiple_attrs_with_same_name(self, dbsession):
             update_attributes=[], 
             add_attributes=[
                 AttrDefSchema(
-                    attr_id=address.id,
+                    name='address',
+                    type='FK',
                     required=True,
                     unique=True,
                     list=True,
                     key=True,
                     bind_to_schema=-1
                 ),
                 AttrDefSchema(
+                    name='address',
+                    type='FK',
                     attr_id=address.id,
```

Comment:
In the above case, you have replaced `attr_id` with `name` and `type`. Here you have added them. I suspect you wanted to replace it here, too?

```suggestion
                    name='address',
                    type='FK',
```